requirements.txt

```diff
@@ -1,3 +1 @@
-# Place any required python libraries here. Will be installed into a pack specific virtualenv
-pycurl==7.19.0
-
+-e git+https://github.com/fmnisme/python-icinga2api.git@9a1a3cc7968d6c72bf49e97ef387b2824e6835e9#egg=icinga2api
```

Comment:
I assume that's because latest version is not published on PyPi (https://pypi.org/project/icinga2api/#history)?

Should try to gently ping the author to publish it to PyPi, although looking at the commit history there don't seem to be that many changes to master since the last release.

Reply:
Yeah, the annoying thing is that the one on PyPi is based on a fork from upstream, but they're not maintaining their fork. I'll post an issue on their repo. Would prefer to get from PyPi.

Reply:
actually...they don't allow posting issues on their repo